docker/xenial-appimage/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update -qq \
     && add-apt-repository -y ppa:beineri/opt-qt-5.14.2-xenial \
     && apt-get -qq update \
     && apt-get -qq upgrade
-RUN apt-get install -y git make build-essential libssl-dev zlib1g-dev libbz2-dev \
+RUN apt-get install -y git make build-essential g++ libssl-dev zlib1g-dev libbz2-dev \
```

Comment:
This is a Dockerimage used to build the AppImage, unrelated to the snap. Did you mean to edit this?